docs/content/en/docs/clustermgmt/etcd-backup-restore/ubuntu-rhel-etcd-backup.md

```diff
@@ -24,6 +24,9 @@ EKS-Anywhere clusters use etcd as the backing store. Taking a snapshot of etcd b
 
 Etcd offers a built-in snapshot mechanism. You can take a snapshot using the `etcdctl snapshot save` command by following the steps given below. 
 
+Note: The following commands use the username as 'ec2-user'. All EKS-A clusters on bare-metal, vSphere, and Snow use the username ‘ec2-user’ for their nodes. For EKS-A CloudStack, use 'capc’ instead of ec2-user; for EKS-A Nutanix, use ‘eksa’ instead of ec2-user.
```

Comment:
We should put this in an alert to make sure users see it. Also we never use "EKS-A" in the docs, it's always EKS Anywhere.

{{% alert title="Note" color="warning" %}}

The following commands use `ec2-user` as the username. For EKS Anywhere on vSphere, Bare Metal, and Snow, the default username is `ec2-user`. For EKS Anywhere on Apache CloudStack, the default username is `capc`. For EKS Anywhere on Nutanix, the default username is `eksa`. 

{{% /alert %}}